index.html

```diff
@@ -41,6 +41,8 @@
                 <a class="dropdown-item" href="https://bluesky.github.io/bluesky-widgets">Bluesky Widgets (beta)&mdash; Graphical Components for Jupyter and Qt</a>
                 <a class="dropdown-item" href="https://bluesky.github.io/suitcase">Suitcase &mdash; Export / Serialization</a>
                 <a class="dropdown-item" href="https://bluesky.github.io/event-model">Event Model &mdash; Data Model & Schema</a>
+                <a class="dropdown-item" href="https://bluesky.github.io/bluesky-queueserver">Bluesky Queue Server &mdash; Server for Remote Control of Experiments</a>
```

Comment:
I think this should move higher in the list, above Tiled. (In fact, there is an entry already there.)

Reply:
I reordered the menu items